src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -603,6 +608,17 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info, @Nullable
 		String[] keywords = info.getKeywords();
 		desc = desc.replace("${element.keywords}", keywords == null ? "" : Joiner.on(", ").join(keywords));
 
+		// Cancellable
+		boolean cancellable = false;
+		for (Class<? extends Event> event : info.events) {
+			if (Cancellable.class.isAssignableFrom(event)) {
```

Comment:
https://github.com/SkriptLang/Skript/blob/6724b012792ef7feab2a9645ede2bbfa46941016/src/main/java/ch/njol/skript/effects/EffCancelEvent.java#LL75C58-L75C58

EffCancelEvent has some additional logic. Maybe just create a static method in that class and call it here?